src/main/scala/com/raphtory/core/analysis/GraphLens.scala

```diff
@@ -73,5 +73,13 @@ final case class GraphLens(
   }
   def vertexVoted(): Unit = voteCount += 1
   def nextStep(): Unit    = superStep += 1
-  def receiveMessage(msg: VertexMessage): Unit = vertexMap(msg.vertexId).receiveMessage(msg)
+  def receiveMessage(msg: VertexMessage): Unit = {
+    try{
```

Comment:
It works but the main question is why this happens? why it can receive a message should have the vertex but the vertex is not there

Reply:
This can be deleted, I thought I removed this after testing.